_posts/2017-10-26-totw-94.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Tip of the Week #94: Callsite Readability and bool Parameters"
+layout: tips
+sidenav: side-nav-tips.html
+published: false
+permalink: tips/94
+type: markdown
+order: 094
```

Comment:
Change to order: "094" (The sorting algorithm converts these to strings for sort order, and was confused. Enclosing them in quotes forces the correct sort comparison.)